src/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/RijndaelImplementation.cs

```diff
@@ -0,0 +1,110 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Security.Cryptography;
+
+namespace Internal.Cryptography
+{
+    internal sealed class RijndaelImplementation : Rijndael
+    {
+        private Aes _impl;
+
+        internal RijndaelImplementation()
+        {
+            LegalBlockSizesValue = new KeySizes[] { new KeySizes(minSize: 128, maxSize: 128, skipSize: 0) };
+
+            // This class shims Aes
+            _impl = Aes.Create();
+            _impl.BlockSize = 128;
+        }
+
+        public override byte[] IV
+        {
+            get { return _impl.IV; }
+            set { _impl.IV = value; }
+        }
+
+        public override byte[] Key
+        {
+            get { return _impl.Key; }
+            set { _impl.Key = value; }
+        }
+
+        public override int KeySize
+        {
+            get { return _impl.KeySize; }
+            set { _impl.KeySize = value; }
+        }
+
+        public override int BlockSize
+        {
+            get { return _impl.BlockSize; }
+            set
+            {
+                // Values which were legal in desktop RijndaelManaged but not here in this shim type
+                if (value == 192 || value == 256)
+                    throw new PlatformNotSupportedException(SR.Cryptography_Rijndael_BlockSize);
+
+                // Any other invalid block size will get the normal �invalid block size� exception.
+                _impl.BlockSize = value;
+            }
+        }
+
+        public override KeySizes[] LegalKeySizes
+        {
+            get { return _impl.LegalKeySizes; }
+        }
+
+        public override CipherMode Mode
+        {
+            get { return _impl.Mode; }
+            set { _impl.Mode = value; }
+        }
+
+        public override PaddingMode Padding
+        {
+            get { return _impl.Padding; }
+            set { _impl.Padding = value; }
+        }
+
+        public override ICryptoTransform CreateEncryptor()
+        {
+            return _impl.CreateEncryptor();
+        }
```

Comment:
Nit: if you wanted, all of these one-liners could be simplified with expression-bodied members, e.g.
```C#
public override ICryptoTransform CreateEncryptor() => _impl.CreateEncryptor();
```